encode.go

```diff
@@ -79,6 +81,12 @@ func NewEncoderFS(w io.Writer, fsys CreateFS) *Encoder {
 	}
 }
 
+// SetIndent sets json encoded data to have provided prefix and indent settings
+func (e *Encoder) SetIndent(prefix string, indent string) {
```

Comment:
I would call it `SetJSONIndent`. The encoder can encode json and raw binary, and indentation does not apply to the binary part.

Reply:
Changed!